### PR TITLE
feat: migrate project body

### PR DESCRIPTION
**Pull request by Jamalam360**
This is not a perfect solution, as Curseforge uses custom HTML where they should really be using standards (like spoilers).
It can be disabled with a `--dont` flag.

### CODE REVIEW

File: API-Curseforge/src/main/kotlin/com.github.p03w.modifold.curseforge_api/CurseforgeAPI.kt [hunk 1]
```diff
@@ -37,6 +34,15 @@ object CurseforgeAPI : APIInterface() {
         }
     }
 
+    fun getProjectDescription(id: Int): String? {
+        return try {
+            getWithoutAuth<CurseforgeDescription>("$root/mods/$id/description").data
+        } catch (ignored: Exception) {
+            ignored.printStackTrace()
+            null
+        }
```
Comment (SilverAndro) [inline API-Curseforge/src/main/kotlin/com.github.p03w.modifold.curseforge_api/CurseforgeAPI.kt]: When does this fail? If it's a legitimate issue the `warn` log method should probably be used as well

File: API-Modrinth/src/main/kotlin/com/github/p03w/modifold/modrinth_api/ModrinthAPI.kt [hunk 2]
```diff
@@ -50,7 +50,7 @@ object ModrinthAPI : APIInterface() {
         }
     }
 
-    const val root = "https://api.modrinth.com/v2"
+    const val root = "https://staging-api.modrinth.com/v2"
```
Comment (SilverAndro) [inline API-Modrinth/src/main/kotlin/com/github/p03w/modifold/modrinth_api/ModrinthAPI.kt]: Don't forget to change this back

**Review by SilverAndro — CHANGES_REQUESTED** [review]

File: API-Modrinth/src/main/kotlin/com/github/p03w/modifold/modrinth_api/ModrinthAPI.kt (hunk 2 above)
Comment (Jamalam360) [reply API-Modrinth/src/main/kotlin/com/github/p03w/modifold/modrinth_api/ModrinthAPI.kt]: Oops, yes

File: API-Curseforge/src/main/kotlin/com.github.p03w.modifold.curseforge_api/CurseforgeAPI.kt (hunk 1 above)
Comment (Jamalam360) [reply API-Curseforge/src/main/kotlin/com.github.p03w.modifold.curseforge_api/CurseforgeAPI.kt]: It fails when the Curseforge API is down, or when the mod isn't found, which in both cases will have been logged by the curse API calls earlier I would assume

Comment (SilverAndro) [reply API-Curseforge/src/main/kotlin/com.github.p03w.modifold.curseforge_api/CurseforgeAPI.kt]: Yeah in both cases you'd get an error back at curseforge project collection

Comment (SilverAndro) [reply API-Curseforge/src/main/kotlin/com.github.p03w.modifold.curseforge_api/CurseforgeAPI.kt]: Well technically not found is a warning but yeah

**Pull request merged**